src/main/java/com/nomiceu/nomilabs/mixin/vanilla/DedicatedServerMixin.java

```diff
@@ -86,7 +83,7 @@ public void changeInitDifficulty(CallbackInfoReturnable<EnumDifficulty> cir) {
                     LabsTranslate.translate(EnumDifficulty.byId(savedDifficulty).getTranslationKey()),
                     LabsTranslate.translate(locked.getTranslationKey()));
 
-            NomiLabs.LOGGER.warn("This is because you are on {} mode!", LabsModeHelper.getFormattedMode());
+            NomiLabs.LOGGER.warn("This is because you are on HOGIFACTORY!");
```

Comment:
"This is because you are on HOGIFACTORY!" -> "This is because HOGIFACTORY forces peaceful mode" or something similar ? The original sentence only really made sense with modes imo